### ISSUE TITLE
Crashing After Finger Shown

### ISSUE THREAD

**Issue by azazelcodes** [GH0STH4CKER/Finger_counter_webcam#1]
Pinky_Tip_x = pixelCoordinatesLandmark[0]
TypeError: 'NoneType' object is not subscriptable
[ WARN:0@4.203] global D:\a\opencv-python\opencv-python\opencv\modules\videoio\src\cap_msmf.cpp (539) `anonymous-namespace'::SourceReaderCB::~SourceReaderCB terminating async callback
![error](https://user-images.githubusercontent.com/68147363/155900104-1264e997-0e13-43f3-baf8-ae90ce295007.png)

**Comment by azazelcodes** [comment]
Update: Cause is when only a PART of the finger is out of frame it crashes

**Comment by GH0STH4CKER** [comment]
@RayCatYT  Yes it crashes when the part of finger is out of the frame. I don't know why its happening.
